src/Functional.Unions.FluentAssertions/Functional.Unions.FluentAssertions.Tests/UnionValueTypeAssertionsTests.cs

```diff
@@ -0,0 +1,72 @@
+using FluentAssertions;
+using System;
+using Xunit;
+
+namespace Functional.Unions.FluentAssertions.Tests
+{
+	public class UnionValueTypeAssertionsTests
+	{
+		public class TOne { }
```

Comment:
To reduce potential confusion with the naming convention associated with generic parameters, rename `TOne` and `TTwo` classes to something else?

``` csharp
// example...  feel free to define your own
public class LotTracked
{
    public string LotCode { get; }
}

public class NotTracked
{
}
```

Reply:
Good suggestion, I could see that getting confusing. Updated in the latest commit!